src/scenes/home/signup/signupContainer.js

```diff
@@ -0,0 +1,26 @@
+import { connect } from 'react-redux';
+import SignUp from './signup'
+import { addEmail, addPassword, addZip } from '../../state/registration/actions'
+
+const mapStateToProps = null;
```

Comment:
Standard practice is to simply not declare `mapStateToProps` if indeed it will be null.